components/d2l-applied-filters/d2l-applied-filters.js

```diff
@@ -216,12 +218,14 @@ class D2lAppliedFilters extends RtlMixin(LocalizeStaticMixin(LitElement)) {
 		const filters = this._selectedEntries && this._selectedEntries.length > 0 ?
 			html`<d2l-labs-multi-select-list
 				collapsable
+				aria-labelledby="d2l-applied-filters-label"
 			>
 				${(this._selectedEntries || []).map((x, index) => html`
 					<d2l-labs-multi-select-list-item
 						text="${x.text}"
 						deletable
 						index="${index}"
+						role="listitem"
```

Comment:
This solves a problem where sometimes NVDA would say "text frame" and nothing else when selecting a filter

Reply:
Ideally, I think `d2l-labs-multi-select-list-item` could set its own role, but I see that it currently doesn't, and `d2l-labs-multi-select-list` [only seems to set it on (all!) children in `connectedCallback`](https://github.com/BrightspaceUILabs/multi-select/blob/master/multi-select-list.js#L159), which means newly-slotted list items wouldn't get the correct role. Since that would affect all consumers, pushing this into the underlying component should improve accessibility everywhere.

So, I think we can leave this out here, and improve the multi-select components so it's handled by default.